Sources/AWSLambdaRuntime/LambdaRuntime.swift

```diff
@@ -15,16 +15,24 @@
 import Logging
 import NIOConcurrencyHelpers
 import NIOCore
+import Synchronization
 
 #if canImport(FoundationEssentials)
 import FoundationEssentials
 #else
 import Foundation
 #endif
 
+// This is our gardian to ensure only one LambdaRuntime is initialized
+// We use a Mutex here to ensure thread safety
+// We use Bool instead of LambdaRuntime<Handler> as the type here, as we don't know the concrete type that will be used
+private let _singleton = Mutex<Bool>(false)
```

Comment:
We can use an Atomic here instead. Singleton is fine for this.

Reply:
OK, will try to use Atomic instead of a Mutex